tethys_apps/base/app_base.py

```diff
@@ -1375,3 +1375,97 @@ def _log_tethys_app_setting_not_assigned_error(cls, setting_type, setting_name):
                         .format(traceback.format_stack(limit=3)[0], setting_type, setting_name,
                                 cls.name.encode('utf-8'))
                         )
+
+    @classmethod
+    def get_templates_path(cls):
+        project_directory = os.path.dirname(sys.modules[cls.__module__].__file__)
+        return os.path.join(project_directory, 'templates', cls.package)
+
+    def check_analytics_config(self):
```

Comment:
Move this logic to tethys_apps/context_processor.py
Embed loaded tags in new block in tethys_apps/templates/tethys_apps/app_base.html